src/components/Scorecard.vue

```diff
@@ -50,21 +50,23 @@
             :key="metric"
```

Comment:
the scorecard is looking really good now - just one issue
<img width="606" alt="Screenshot 2022-04-21 at 11 03 09 AM" src="https://user-images.githubusercontent.com/13185699/164381434-e24c5cc7-0971-47d6-8455-56dd4cc612af.png">